com.avaloq.tools.ddk.xtext/src/com/avaloq/tools/ddk/xtext/resource/AbstractSelectorFragmentProvider.java

```diff
@@ -38,6 +42,8 @@
   @Inject
   private ShortFragmentProvider shortFragmentProvider;
 
+  private final ConcurrentMap<Object, Object> eclassToCaseSensitive = Maps.newConcurrentMap();

```

Comment:
don't know whether caching will help a lot here... i guess a usual hashmap will do here if you want to keep it

Reply:
Done